### ISSUE TITLE
Module plots hard to work with 

### ISSUE THREAD

**Issue by max-schofield** [trophia/analyser#8]
grid.arrange()  creates a gtable object that I haven't found a successful way to modify when called within my knitr scripts. I've been trialing options from the grid, gridDebug and gridExtra packages but not successfully yet... 
I think it's desirable to have customisable plots (like ggplot style) because it allows more tailored applications, The solution I see at the moment would be to let the modules generate plots individually and then the user can arrange them and modify them as need be (like what is possible with: corer$bubble_plot()).

**Comment by nokome** [comment]
Good feedback, I'll change to returning a list of plots which you can then arrange as wanted. Let's see how that goes.

**Comment by nokome** [comment]
This has been addressed in commit 0ed35b30a7f754133ef8b422a69dc4b7a7f2a781. Now, functions that produce multiple plots return a list of plots that can then be customised by the caller e.g.

```
      list(
        effort = plot_effort,
        catch = plot_catch
      )
```

To make it clear that a function returns a list there has been some renaming e.g. `rollup_plot` -> `rollup_plots`

**Comment by max-schofield** [comment]
Cool, looks like that latest run failed to find analyser

On 14 June 2016 at 09:34, Nokome Bentley notifications@github.com wrote:

> Closed #8 https://github.com/trophia/analyser/issues/8.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> https://github.com/trophia/analyser/issues/8#event-691053707, or mute
> the thread
> https://github.com/notifications/unsubscribe/AMsMagYfNdx7j5ZROIbOExlVxu19QYPCks5qLczQgaJpZM4Iz7Ai
> .

## 

[image: TridentSystems_logo_circle] http://www.tridentsystems.co.nz/

_Max Schofield_

_Fisheries Analyst_

DDI:        +64 4 385 4005 extension 1508

Mobile:     +64 27 326 6926

_max.schofield@tridentsystems.co.nz max.schofield@tridentsystems.co.nz_

www.tridentsystems.co.nz

Level 6, 135 Victoria Street,
https://www.google.co.nz/maps/place/135+Victoria+St,+Te+Aro,+Wellington+6011/

Wellington 6011, NZ
https://www.google.co.nz/maps/place/135+Victoria+St,+Te+Aro,+Wellington+6011/

PO Box 297,

Wellington 6140, NZ

_Disclaimer: This e-mail message and any attachments are privileged and
confidential. They may contain information that is subject to statutory
restrictions on their use. If you are not the intended recipient you must
not use, distribute or copy this email. If you have received this email in
error please notify the sender immediately and delete this email. Any views
expressed in this email are not necessarily the views of Trident Systems._
